docs/source/conf.py

```diff
@@ -93,6 +93,13 @@
 sphinx_gallery_conf = {
     'capture_repr': (),
     'examples_dirs': ['../../examples'],
+    # NASA wmts servers are returning bad content metadata
+    "expected_failing_examples": [
+        '../../examples/scalar_data/eyja_volcano.py',
```

Comment:
It seems like these two are flaky right now. I get failures when I remove them. I get failures when I add them :) Not sure if there is a way to say something is flaky in sphinx gallery?
```suggestion
        '../../examples/scalar_data/eyja_volcano.py',
        '../../examples/web_services/image_tiles.py',
```

Reply:
Apparently the Stamen tiles have moved https://stackoverflow.com/questions/77248120/stamen-terrain-map-not-working-in-cartopy

Reply:
Interesting, I did not realize that was going away. Let's follow that up in a separate PR then and leave the docs failing here for now.